README.md

```diff
@@ -1,45 +1,272 @@
-# GitHub Safe Settings
+# GitHub Safe-Settings
 
 [![Node.js CI](https://github.com/github/safe-settings/actions/workflows/node-ci.yml/badge.svg)](https://github.com/github/safe-settings/actions/workflows/node-ci.yml)
 [![Dependabot][dependabot-badge]][dependabot-link]
 
-This is a modified version of [Settings Probot](https://github.com/probot/settings) GitHub App. This differs from the original probot settings app in several ways:
+`Safe-settings`– an app to manage policy-as-code and apply repository settings to repositories across an organization.
 
-1. It does not use [probot-config](https://github.com/probot/probot-config). Instead, it reads the settings from `.github/settings.yml` file contained in the `admin` repo in the organization. The `admin` repo should be a restricted repository and contains the settings for all the repos within the organization.
-1. It manages the settings for all the repositories in the organization. Repositories could be explicitly defined in the settings config, but the app also manages any repo that is created in the organization.
-1. It will allow you to set branch protections on the `default` branch no matter what it is called by calling it `default` in `.github/settings.yml`.
+It is loosely based on [Settings Probot](https://github.com/probot/settings) GitHub App. However, it differs from the original probot settings app in several important ways:
 
-## Usage
+1. In `safe-settings` all the settings are stored centrally in an `admin` repo within the organization. 
+1. There are 3 levels at which the settings could be managed:
+   1. Org-level settings are defined in `.github/settings.yml` 
+   1. The org-level settings could be overridden at a  `suborg` level. These would be a collection of repos belonging to a project or a business unit, or a team. The `suborg`settings reside in the `.github/suborgs`folder.
+   1. Lastly, we could override settings for individual repos by creating a repo specific yaml in `.github/repos`folder
+1. `Safe-settings` explicitly looks in the `admin` repo in the organization for the settings files. The `admin` repo should be a restricted repository with `branch protections` and `codeowners` . 
+1. To address the scalability concerns for large organizations having thousands of repos, it is recommended to break the settings into org-level, suborg-level, and repo-level units. This will allow different teams to be define and manage policies for their specific projects or business units.
+
+
+
+## Table of Contents
+[TOC]
```

Comment:
Perhaps this should just be removed for now...

Reply:
Totally makes sense. The intent was it would be auto-generated but that didn;t pan out.